docs/en/ingest-management/security/mutual-tls.asciidoc

```diff
@@ -0,0 +1,200 @@
+
+To facilitate the bootstrapping process and enable {agent} to establish an mTLS connection with {fleet-server}, all certificates and certificate authorities are configured using command-line parameters during the agent installation. Once the mTLS connection between {agent} and the {fleet-server} is established, the policy configuration enables the establishment of the mTLS connection between {agent} and the designated output as well.
+
+[discrete]
+[[mutual-tls-on-premise]]
+== On-premise deployments
+
+image::images/mutual-tls-on-prem.png[Diagram of mutual TLS on premise deployment model]
+
+Refer to the steps in <<secure-connections,Configure SSL/TLS for self-managed {fleet-server}s>>. To configure mutual TLS, include the following additional parameters when you install {agent} and {fleet-server}.
+
+[discrete]
+=== {agent} settings
+During {agent} installation on premise use the following options:
+
+[cols="1,1"]
+|===
+|`-certificate-authorities`
+|List of CA certificates that are trusted when {fleet-server} connects to {agent} 
+
+|`-elastic-agent-cert`
+|{agent} certificate to present to {fleet-server} during authentication
+
+|`-elastic-agent-cert-key`
+|{agent} certificate key to present to {fleet-server}
+|===
+
+[discrete]
+=== {fleet-server} settings
+During {fleet-server} installation on-premise {fleet-server} authenticates with {es} and {agents}. You can use the following CLI options to facilitate these secure connections:
+
+[cols="1,1"]
+|===
+|`-fleet-server-es-ca`
+|CA to use for the {es} connection 
+
+|`-fleet-server-es-cert`
+|{fleet-server} certificate to present to {es}
+∫
+|`-fleet-server-es-cert-key`
+|{fleet-server} certificate key to present to {es}
+
+|`-certificate-authorities`
+|List of CA certificates that are trusted when {agent} connects to {fleet-server}
+
+|`-fleet-server-cert`
+|{fleet-server} certificate to present to {agents} during authentication
+
+|`-fleet-server-cert-key`
+|{fleet-server} certificate key to present to {agents}  
+|===
+
+[discrete]
+=== {fleet} settings:
+
+In {kib}, navigate to {fleet}, open the **Settings** tab, and choose the **Output** that you'd like to configure.
+In the **Advanced YAML configuration**, add the following settings:
+
+[cols="1,1"]
+|===
+|`-certificate-authorities`
```

Comment:
[Blocker]
the correct key is:
```suggestion
|`ssl.certificate-authorities`
```

Reply:
I've changed all instances of `-certificate-authorities` to `ssl.certificate-authorities`.